.ci/travis/travis.sh

```diff
@@ -69,6 +69,7 @@ no-error-test-sbe)
   mvn clean install -Pno-validations
   git clone https://github.com/real-logic/simple-binary-encoding.git
   cd simple-binary-encoding
+  git checkout $(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
no idea, but they could do what ever they want for their dev process.
I will use special commit (https://github.com/checkstyle/checkstyle/commit/724fcc53be66da409d51eb4e176b15b2727692a4) before latest commit to SNAPSHOT dependency, as they upgraded to checkstyle8.1 recently.